### PR TITLE
Test case for `internal/params`

### PR DESCRIPTION
**Pull request by kevindiu**
Package internal/param test case
### Description:

Refactor params.go for testability and param test case. 

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [x] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [x] I have added tests and benchmarks to cover my changes.
- [x] I have ensure all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by hlts2** [comment]
Is this PR in the process of working? ?:thinking:
I want an test for `option.go`. :bow::bow::bow:

**Comment by kevindiu** [comment]
yes, it is in progress.

File: internal/params/option_test.go [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+				if got.filePath.defaultPath != "path" {
+					return errors.New("invalid param was set")
+				}
+				return nil
+			},
+		},
+
+		{
+			name: "not set when path is empty",
+			path: "",
```
[the remark this answers is not in the document]
Comment (hlts2) [reply internal/params/option_test.go]: thank you for your review.
I fixed

**Comment by vankichi** [comment]
It is no longer updated.
We should do this task as a new PR.

**Pull request closed without merging**